includes/components/class-admin.php

```diff
@@ -946,6 +949,261 @@ protected function get_extensions_tab() {
+						],
+						[
+							'label' => esc_html__( 'WordPress version:', 'hivepress' ),
+							'value' => get_bloginfo( 'version' ),
+						],
+						[
+							'label' => esc_html__( 'WordPress multisite:', 'hivepress' ),
+							'value' => is_multisite() ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'WordPress memory limit:', 'hivepress' ),
+							'value' => WP_MEMORY_LIMIT,
+						],
+						[
+							'label' => esc_html__( 'WordPress debug mode:', 'hivepress' ),
+							'value' => ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'WordPress cron:', 'hivepress' ),
+							'value' => ! ( defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'Language:', 'hivepress' ),
+							'value' => get_locale(),
+						],
+						[
+							'label' => esc_html__( 'External object cache:', 'hivepress' ),
+							'value' => wp_using_ext_object_cache() ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+					],
+				],
+				[
+					'label'  => esc_html__( 'Server environment', 'hivepress' ),
+					'values' => [
+						[
+							'label' => esc_html__( 'Server info:', 'hivepress' ),
+							'value' => isset( $_SERVER['SERVER_SOFTWARE'] ) ? wc_clean( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'PHP version:', 'hivepress' ),
+							'value' => phpversion(),
+						],
+						[
+							'label' => esc_html__( 'PHP post max size:', 'hivepress' ),
+							'value' => ini_get( 'post_max_size' ),
+						],
+						[
+							'label' => esc_html__( 'PHP time limit:', 'hivepress' ),
+							'value' => (int) ini_get( 'max_execution_time' ),
+						],
+						[
+							'label' => esc_html__( 'PHP max input vars:', 'hivepress' ),
+							'value' => (int) ini_get( 'max_input_vars' ),
+						],
+						[
+							'label' => esc_html__( 'cURL version:', 'hivepress' ),
+							'value' => $curl_version,
+						],
+						[
+							'label' => esc_html__( 'SUHOSIN installed:', 'hivepress' ),
+							'value' => extension_loaded( 'suhosin' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'MySQL version:', 'hivepress' ),
+							'value' => $wpdb->db_server_info(),
+						],
+						[
+							'label' => esc_html__( 'Max upload size:', 'hivepress' ),
+							'value' => size_format( wp_max_upload_size() ),
+						],
+						[
+							'label' => esc_html__( 'Default timezone:', 'hivepress' ),
+							'value' => date_default_timezone_get(),
+						],
+						[
+							'label' => esc_html__( 'fsockopen/cURL:', 'hivepress' ),
```

Comment:
It's ok to leave such strings non-translatable, translation will not be needed for sure. You can also check and reduce other strings to hard-coded ones, e.g. instead of __("Something Enabled/Available") just "Something" with yes/no icons